js/index.js

```diff
@@ -5,6 +5,10 @@
 
   var tutorialContainer = document.body;
 
+  var smartButtons = Array.prototype.slice.call(
```

Comment:
Please add `XXX: These buttons come from smart-button web components but they are supported only for certified apps in v2.5 . We should change them back to web components once we can use them.`